juice/Cargo.toml

```diff
@@ -28,6 +28,7 @@ rand = "0.7"
 num = "0.2"
 capnp = "0.13"
 timeit = "0.1"
+transpose = "*"
```

Comment:
we should avoid these, one cannot release to crates-io with that kind of dependency

Reply:
We will! It's going to be stripped completely and replaced with the CUBLAS tranpose. This is just quicker to test.